runtime/src/test/java/jdocs/akka/grpc/client/GrpcClientSettingsCompileOnly.java

```diff
@@ -23,12 +26,13 @@ public static void sd() {
                 .withTls(false);
         //#simple-programmatic
 
+        SimpleServiceDiscovery serviceDiscovery= ServiceDiscovery$.MODULE$.apply(actorSystem).discovery();
+
         //#provide-sd
         // An ActorSystem's default service discovery mechanism
-        GrpcClientSettings.discoverService(
+        GrpcClientSettings.connectTo(
                 "my-service",
-                443,
-                "config", // config based service discovery must be defined in the ActorSystems' config
+                serviceDiscovery,
```

Comment:
Hmm, now we're passing in both `ServiceDiscovery` and the `ActorSystem` - would it make sense to (also) have a `connectTo` that only takes the service name(s) and the `ActorSystem`?

Reply:
I like the simplification, but then I'd rename the method to `GrpcClientSettings.usingServiceDiscovery(String, ActorSystem)` (`usingServiceDiscovery` isntead of `connectTo`) so it's clear why only a `String` is necessary.